pkg/maps/lbmap/lbmap.go

```diff
@@ -548,3 +548,16 @@ func ServiceValue2LBBackEnd(svcKey ServiceKey, svcValue ServiceValue) (*loadbala
 
 	return loadbalancer.NewLBBackEnd(loadbalancer.TCP, feIP, fePort, feWeight)
 }
+
+// DeleteRevNATBPF deletes the revNAT entry from its corresponding BPF map
+// (IPv4 or IPv6) with ID id. Returns an error if the deletion operation failed.
+func DeleteRevNATBPF(id loadbalancer.ServiceID, isIPv4 bool) error {
+	var revNATK RevNatKey
+	if isIPv4 {
```

Comment:
Most, if not all functions have an `isIPv6` instead of `isIPv4`. Consider change the function's argument to have `isIPv6` instead.

Reply:
Fixed in next upload.

Reply:
Fixed, and ensured that callers were changed to account for this swapping of logic.